python/lsst/ts/Dome/mock_controller.py

```diff
@@ -247,13 +247,16 @@ async def move_az(self, **kwargs):
         ----------
         kwargs: `dict`
             A dictionary with arguments to the function call. It should contain the key "azimuth" with a
-            float value.
+            float value and the key "azRate" with a signed float value where positive means towards
```

Comment:
Please consider deleting "signed" from "signed float value" here and below.

Reply:
Can `azimuth` be signed? If it has a required range this seems like a good place to state it. (I assume you will match the vendor's code, but I have no idea what it does).

Reply:
Done.